source/lightspeed/lightSpeedInLeft.css

```diff
@@ -0,0 +1,24 @@
+@keyframes lightSpeedInLeft {
+  from {
+    transform: translate3d(-100%, 0, 0) skewX(30deg);
+    opacity: 0;
+  }
+
```

Comment:
Any codepen link to those new animations?

Reply:
I will prepare one for you

Reply:
Here's the link: https://codepen.io/warengonzagaofficial/full/mdJayJG @eltonmesquita